golang.mk

```diff
@@ -65,17 +65,17 @@ _build: vendor
 		$(TARGET);\
 	)
 
-build: _build
+build: go_generate _build
 	$(foreach TARGET,$(TARGETS),ln -sf $(OUTPUT_FILE) dist/$(OUTPUT_LINK);)
 
 compress: _build
 	tar czf dist/$(OUTPUT_FILE).tar.gz dist/$(OUTPUT_FILE)
 
-xc:
+xc: go_generate
 	GOOS=linux GOARCH=amd64 make compress
 	GOOS=darwin GOARCH=amd64 make compress
 
-install: vendor test
+install: test vendor
```

Comment:
Well, that's weird now. A test won't work without vendoring before. Either the target naming is wrong now, or I won't work on a clean machine.

Reply:
After investigating (because it's running fine on a clean machine) I noticed that `test` is running `test_packages` which itself is running `vendor`, so it's actually not used in the install step.